refinery/analysis_manager/tests.py

```diff
@@ -176,18 +176,6 @@ def test_fetch_objects_required_for_analyses_bad_user(self):
             )
             self.assertIn("Couldn't fetch User", context.exception.message)
 
-    def test_fetch_objects_required_for_analyses_bad_dataset(self):
```

Comment:
discussed w/ Scott about removing this unit test since it tests for an error throw outside this function